js/digifinex.js

```diff
@@ -331,8 +372,12 @@ module.exports = class digifinex extends Exchange {
 
     async fetchMarketsV2 (params = {}) {
         const defaultType = this.safeString (this.options, 'defaultType');
-        const method = (defaultType === 'margin') ? 'publicGetMarginSymbols' : 'publicGetTradesSymbols';
-        const response = await this[method] (params);
+        const [ marginMode, query ] = this.handleMarginModeAndParams ('fetchMarketsV2', params);
+        let method = (marginMode !== undefined) ? 'publicSpotGetMarginSymbols' : 'publicSpotGetTradesSymbols';
+        if (defaultType === 'swap') {
+            method = 'publicSwapGetPublicInstruments';
+        }
+        const response = await this[method] (query);
```

Comment:
@Dan-krm why can't we load swap+spot market simultaneously? Apparently, they have different Ids, am I missing something?

Reply:
@Dan-krm can you check this comment? ⬆️

Reply:
Hey I just noticed this comment now, I think the way it was handled was because of restraints with the previous implementation, I've updated it to load the swap + spot/margin markets at the same time